### PR TITLE
ShiftIt upgrade to 1.6.1

### PR DESCRIPTION
**Pull request by fikovnik**

### CODE REVIEW

File: Casks/shiftit.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 cask :v1 => 'shiftit' do
-  version '1.6'
-  sha256 '538e0628d6fc99c3562694c0722cb41699d6a802f0032678ef05fa2b53711955'
+  version '1.6.1'
+  sha256 '64b9fdb4b8104a3e8efc307119ca4caba68a983916a4424db507b1e05cec3644'
 
-  url "https://github.com/downloads/fikovnik/ShiftIt/ShiftIt-develop-#{version}.zip"
+  url "https://github.com/fikovnik/ShiftIt/releases/download/version-#{version}/ShiftIt-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/shiftit.rb]: Could you add a `url "'ttps://github.com/fikovnik/ShiftIt/releases.atom'` line under this one?

**Comment by adius** [comment]
The appcast sha256 must be update as well, right?
It is  31b00662480244b9c2a739e342154fbfa52e50d29bd9c3a4fd683e6f7e905a04

**Comment by vitorgalvao** [comment]
@adius Appcasts aren't yet functional, so it doesn’t matter for now.

File: Casks/shiftit.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/shiftit.rb]: Apologies, I meant `appcast`, not `url`. Will fix in a different pull request. Your contribution will still be included (and still credited to you), with the appropriate modifications.

Thank you for the contribution.

**Pull request closed without merging**